frontend/src/components/auth/components/sign-up-form/sign-up-form.module.scss

```diff
@@ -1,41 +1,41 @@
-.signUpTitle,
-.signUpForm,
-.signInLink,
-.signUpFormPolicy,
-.signUpLogo {
-  width: 25%;
-  margin: 0 auto;
+.signUpWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 476px;
+  height: 657px;
+  padding: 122px 102px;
```

Comment:
Please avoid using absolute values in px. In this case you can use `width: N%; min-width: 450px`. If you really need absolute values, pelase round them: 476 => 470 or 450.